### PR TITLE
Clarify help text and error messages for invalid <dir> values

### PR DESCRIPTION
**Pull request by johnbillion**
This improves the help text and adds clearer error messages relating to the `<dir>` flag.

### CODE REVIEW

File: src/ScaffoldPackageCommand.php [hunk 1]
```diff
@@ -522,8 +531,12 @@ public function package_tests( $args, $assoc_args ) {
...
 
-		if ( ! is_dir( $package_dir ) || ! file_exists( $package_dir . '/composer.json' ) ) {
-			WP_CLI::error( "Invalid package directory. composer.json file must be present." );
+		if ( ! is_dir( $package_dir ) ) {
+			WP_CLI::error( 'Directory does not exist.' );
+		}
+
+		if ( ! file_exists( $package_dir . '/composer.json' ) ) {
+			WP_CLI::error( 'Invalid package directory. composer.json file must be present.' );
 		}
```
Comment (danielbachhuber) [inline src/ScaffoldPackageCommand.php]: Given the repetition of code, what do you think about creating `self::check_package_dir( $package_dir )`, and abstracting these error checks to it?

**Comment by danielbachhuber** [comment]
I've enabled the setting for Circle to build forked pull requests too.

**Comment by johnbillion** [comment]
Something's up with Circle CI

**Comment by danielbachhuber** [comment]
> Something's up with Circle CI

@johnbillion You'll need to push first, and then it will build the next push.

**Pull request merged**